sdcm/cluster.py

```diff
@@ -543,7 +543,7 @@ def download_prometheus_data_dir(self):
         self.remoter.receive_files(src=self.prometheus_data_dir, dst=dst)
 
     def _write_prometheus_cfg(self, targets):
-        targets_list = ['%s:9103' % ip for ip in targets]
+        targets_list = ['%s:9103' % str(ip) for ip in targets]
```

Comment:
That is the kind of thing that I didn't think it would be necessary. Any particular bug you are trying to fix with this?